backend/src/main/java/com/redhat/cloud/notifications/routers/InternalService.java

```diff
@@ -282,6 +298,92 @@ public void httpRoot() {
         .onFailure().recoverWithItem(throwable -> Response.status(Response.Status.BAD_REQUEST).entity(new RenderEmailTemplateResponse.Error(throwable.getMessage())).build());
     }
 
+    @POST
+    @Path("/behaviorGroups/default")
+    @Consumes(APPLICATION_JSON)
+    @Produces(APPLICATION_JSON)
+    public Uni<BehaviorGroup> createDefaultBehaviorGroup(@NotNull @Valid BehaviorGroup behaviorGroup) {
+        return sessionFactory.withSession(session -> {
+            return behaviorGroupResources.createDefault(behaviorGroup);
+        });
+    }
+
+    @PUT
+    @Path("/behaviorGroups/default/{id}")
```

Comment:
I didn't see a test for this, but maybe I missed it.

Reply:
There wasn't any, but the db code is now tested